### ISSUE TITLE
Driver for Jetson TX2 NX

### ISSUE THREAD

**Issue by jmadeira-pdm** [RidgeRun/NVIDIA-Jetson-IMX477-RPIV3#37]
Hello,

I am currently working on porting this patch for the TX2 NX, but I have a question regarding the files affected by this patch.

Namely I cannot locate the file imx477.c on the TX2 NX kernel files, is this file supplied somewhere else?

**Comment by dustinkerstein** [comment]
@jmadeira-pdm, were you able to find these sources and build a working driver for the TX2 NX? Thanks!

**Comment by Mark-Bezyinski** [comment]
> @jmadeira-pdm, were you able to find these sources and build a working driver for the TX2 NX? Thanks!

@dustinkerstein 

Hello!
Any updates so far?
Could you help me please connect IMX477 to TX2 NX?

Mark

**Comment by Bra1nsen** [comment]
Yes we are also interested in using Sony IMX477 Raspberry HQ Cam with TX2 Module. Any updates?
